libs/damap/src/lib/widgets/person-search/person-search.component.html

```diff
@@ -12,7 +12,30 @@
       <mat-option
         *ngFor="let person of result"
         (onSelectionChange)="selectPerson(person)">
-        {{ person.firstName }} {{ person.lastName }}
+        <div class="person-container">
+          <span class="person-name"
+            >{{ person.firstName }} {{ person.lastName }}</span
+          >
+          <div
+            *ngIf="
+              person.mbox ||
+              (person.personId?.identifier &&
+                person.personId?.type === identifierType.ORCID)
+            "
+            class="person-details">
+            <span *ngIf="person.mbox" class="person-info">{{
+              person.mbox
+            }}</span>
+            <span
+              *ngIf="
+                person.personId?.identifier &&
+                person.personId?.type === identifierType.ORCID
```

Comment:
It may be any identifier so just checking for ORCID is a bit restrictive. If there is an identifier, it would be good to just display it as: `<identifier type>:<identifier value>`. So this check and the import in the `.ts` file can be removed.